lib/rules/declaration-property-value-no-unknown/index.mjs

```diff
@@ -46,6 +47,8 @@ const rule = (primary, secondaryOptions) => {
 					ignoreProperties: [validateObjectWithArrayProps(isString, isRegExp)],
 					propertiesSyntax: [validateObjectWithProps(isString)],
 					typesSyntax: [validateObjectWithProps(isString)],
+					atRulesSyntax: [validateAtRulesSyntax],
```

Comment:
Most of our `RuleOptionsPossibleFunc` functions are rudimentary and not specialized.
i.e. can be part of utils

I am not sure if we want to start a trend which introduces validators that deeply check the shape of objects.
I don't have a strong opinion because we have `--validate` to balance any hypothetical performance hit.
I would prefer #7608 though.

Reply:
@ybiquitous what's your opinion?